community/distcc/build

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+# Running distcc as a server instead of a client
+# requires python to be installed
```

Comment:
Full stop `.`

Reply:
@dylanaraps Should I add and resend a PR?

Reply:
Oops. I merged it anyway didn't I? That's my bad, apologies. If you like, we can fix this when the next release of `distcc` comes out.

Reply:
Up to you!

Reply:
distcc doesn't make releases often, so I'll send a PR now to not forget about it later :P

Reply:
Sounds good. :+1: